ios/Helio.xcworkspace/contents.xcworkspacedata

```diff
@@ -2,7 +2,7 @@
 <Workspace
    version = "1.0">
    <FileRef
-      location = "group:BetterSocial.xcodeproj">
+      location = "group:Helio.xcodeproj">
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `BetterSocial.xcodeproj` to `Helio.xcodeproj` is consistent with the PR title and description, which mention a name change for the app. Ensure that the new project file `Helio.xcodeproj` exists and all references to the old project file `BetterSocial.xcodeproj` have been updated throughout the codebase to avoid any build or runtime issues.
```diff
-      location = "group:BetterSocial.xcodeproj">
+      location = "group:Helio.xcodeproj">
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->